geth/rpc/call_raw.go

```diff
@@ -56,6 +69,8 @@ type jsonError struct {
 	Data    interface{} `json:"data,omitempty"`
 }
 
+type contextKey string
```

Comment:
Can we move it to the top of this file before `const` in which it is used?

Reply:
Will update this before landing